tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py27,py33,py34,pypy
+envlist=py27,py36,pypy
```

Comment:
Hi @macbeth322,
You probably missed `py37` here.

Reply:
No... 37 requires annoying changes to the Travis config I have not gotten around to yet

Reply:
I suppose you can just add `dist: xenial` into `.travis.yml`.

Reply:
By the way, how is `tox.ini` relevant to `Travis`?

Reply:
Yeah the config I need to do is similar to what I set up here https://github.com/Bachmann1234/Thrift-Explorer/blob/master/.travis.yml I just have not gotten around to it

Reply:
Honestly reading it now its less annoying then I remember, ill get to it soon. Less relevant to this PR. If it breaks 3.7 ill deal with it

Reply:
I just created a PR to add Python 3.7 for the corresponding branch:
macbeth322/marshmallow-polyfield#2

Reply:
I've never used travis personally, but if @Bachmann1234 thinks macbeth322#2 looks good I can merge it.

Reply:
Looks good to me